python/beeai_framework/adapters/litellm/chat.py

```diff
@@ -67,6 +68,15 @@ def __init__(self, model_id: str, settings: dict | None = None) -> None:
         self.settings = settings or {}
         super().__init__()
 
+    @staticmethod
+    def litellm_debug(enable: bool = True) -> None:
+        litellm.set_verbose = enable
+        litellm.suppress_debug_info = not enable
+        litellm.logging = enable
+
+        logger = logging.getLogger("LiteLLM")
+        logger.setLevel(logging.CRITICAL + 1 if enable else logging.DEBUG)
```

Comment:
inverse of intention?

Reply:
Also how will this interact with [LITELLM_LOG](https://docs.litellm.ai/docs/proxy/debugging) ? I presume it will override. Could we also respect that env var? Alternatively initialize off a BEE env var otherwise?

Reply:
updated, user can update the behaviour by calling this method, env configuration can be further request